### PR TITLE
[android] disable click event when gesture already consume touch event

### PR DESCRIPTION
**Pull request by sospartan**
Disable click event when gesture is consumed same serial touch-event.

### CODE REVIEW

File: android/sdk/src/main/java/com/taobao/weex/ui/view/gesture/WXGesture.java [hunk 1]
```diff
@@ -309,12 +320,14 @@ public boolean onTouch(View v, MotionEvent event) {
...
           break;
         case MotionEvent.ACTION_UP:
         case MotionEvent.ACTION_POINTER_UP:
           finishDisallowInterceptTouchEvent(v);
           result |= handleMotionEvent(LowLevelGesture.ACTION_UP, event);
           result |= handlePanMotionEvent(event);
+          mIsTouchEventConsumed = mIsTouchEventConsumed || result;
           break;
         case MotionEvent.ACTION_CANCEL:
           finishDisallowInterceptTouchEvent(v);
```
Comment (YorkShen) [inline android/sdk/src/main/java/com/taobao/weex/ui/view/gesture/WXGesture.java]: Reset flag to false when it is canceled?

Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/ui/view/gesture/WXGesture.java]: It's not necessary to reset flag here,`Pan` may already trigger by 'move' .

Comment (YorkShen) [inline android/sdk/src/main/java/com/taobao/weex/ui/view/gesture/WXGesture.java]: Reset to false when **cancel** fired?

Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/ui/view/gesture/WXGesture.java]: click event should not trigger when any of gesture events (start/move)  fired. So `ACTION_CANCEL` will not reset flag.

**Pull request merged**